### DISCUSSION TITLE
Button alignment

### DISCUSSION THREAD

**Discussion by cavaughankirov** [thehighnibble/thehighnibble.github.io#97] [S-132 add-on for IMSAI & Cromemco replicas]
Worried about how I installed the 2 push buttons. On the picture it shows the writing on the buttons (directions based on the pic) with one  (reset) to the left and the other (prog) to the bottom. Both of mine are to the top. I noticed that for both of mine the circuit is complete from top to bottom on mine. This might be fine for prog in my case, but wrong for reset.

**Comment by ronaldlw** [comment]
Hopefully you've already figured this out, but for any newcomers that might have the same question:

Assuming you're talking about these buttons:
![image](https://github.com/user-attachments/assets/e8adc545-a864-4615-9e9d-5c58df58e177)

They will only go in one of two ways, and either way should be fine.  I noticed on mine the writing isn't always on the same side for whatever reason.
